docs/contributing/checklist.md

```diff
@@ -0,0 +1,167 @@
+---
+id: checklist
+title: Contribution Checklist
+---
+
+This document includes a checklist that summarizes the list of files that you need in order to contribute to the Cortex XSOAR content repository. Please make sure you have everything before you open a Pull Request (unless stated otherwise).
+
+Keep in mind that content packs can contain multiple types of entities, such as Integrations, Automations, Playbooks, Incident Types, Incident Fields, and so on.
+
+Depending on whether your content pack is aiming to be *certified* or not, there will be different requirements.
+
+This article also includes a Pull Request checklist, that summarized everything you must do before and after opening a Pull Request on GitHub to contribute your pack.
+
+Happy contributing!
+
+# Content Pack checklist
+
+No matter you include in your content pack or whether it's going to be certified, the pack must include the following:
+
+- Pack Metadata file (i.e. `Packs/YourPackName/pack_metadata.json`) : the information about your content pack. It should be compiled with all the required information, as explained [here](packs-format).
+- Pack README (i.e. `Packs/YourPackName/README.md`): the readme of the pack file. More information [here](pack-docs).
+- Release Notes (i.e. `Packs/YourPackName/ReleaseNotes/1_0_1.md`): these are required only if you are updating an existing Content pack, not for the first. More information [here](release-notes).
+
```

Comment:
Are we planning to move all three of these docs under contributing section? Right now there are under `integrations/packs-format` path and under `Getting Started` in the sidebar.

Reply:
Let me fix the links first, we'll see the reorganization tomorrow.